### PR TITLE
Add scenario to LOADHISTORY section

### PR DESCRIPTION
**Pull request by erinstellato-ms**
This additional paragraph is added after a user ran the restore with LOADHISTORY command for a set of backup files in the incorrect order (e.g. full backup, tlog backup 2, tlog backup 3, tlog backup 1).  The restore sequence is generated based on the backup_set_id (SMO does not interrogate LSN), so the restore sequence was incorrect.

### CODE REVIEW

**Comment by PRMerger17** [comment]
@erinstellato-ms : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by erinstellato-ms** [comment]
@MikeRayMSFT feel free to edit/improve

**Comment by Jak-MS** [comment]
@MikeRayMSFT 

Can you review this PR?

IMPORTANT: When this content is ready to merge, you must add `#sign-off` in a comment or the approval may get overlooked.


#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Review by MikeRayMSFT — APPROVED** [review]

**Comment by erinstellato-ms** [comment]
#sign-off

**Comment by PRMerger10** [comment]
Invalid command: '#sign-off'. Only the assigned author of one or more file in this PR can sign off. @MikeRayMSFT

File: docs/t-sql/statements/restore-statements-arguments-transact-sql.md [hunk 1]
```diff
@@ -216,6 +216,8 @@ DATABASE_SNAPSHOT **=**_database\_snapshot\_name_
...
+ 
+Be aware that using LOADHISTORY for backups which already exist in the `msdb` history tables will add the same information with a new backup_set_id.  Further, if you use LOADHISTORY to recreate backup history in `msdb`, either on a different server or after it was deleted from the original server, it is recommended to run the restore commands for the backups in the order in which they were taken.  This ensure the LSN chain remains intact and the SSMS restore wizard will correctly read the backup history to generate the correct restore sequence.  Use of LOADHISTORY with backup history recreated out of order can result in an error when trying to restore (“Unable to create restore plan due to break in the LSN chain. (Microsoft.SqlServer.SmoExtended)”).
```
Comment (Jak-MS) [inline docs/t-sql/statements/restore-statements-arguments-transact-sql.md:220]:
```suggestion
Be aware that using LOADHISTORY for backups which already exist in the `msdb` history tables will add the same information with a new backup_set_id.  Further, if you use LOADHISTORY to recreate backup history in `msdb`, either on a different server or after it was deleted from the original server, it is recommended to run the restore commands for the backups in the order in which they were taken.  This ensures the LSN chain remains intact and the SSMS restore wizard will correctly read the backup history to generate the correct restore sequence.  Use of LOADHISTORY with backup history recreated out of order can result in an error when trying to restore (“Unable to create restore plan due to break in the LSN chain. (Microsoft.SqlServer.SmoExtended)”).

```

**Comment by PRMerger5** [comment]
@Jak-MS : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**